lib/service/notification.dart

```diff
@@ -17,22 +17,42 @@ class AlarmNotification {
 
   /// Adds configuration for local notifications and initialize service.
   Future<void> init() async {
-    const initializationSettingsAndroid =
-        AndroidInitializationSettings('@mipmap/ic_launcher');
-    const initializationSettingsIOS = DarwinInitializationSettings(
+    const initializationSettingsAndroid = AndroidInitializationSettings(
+      '@mipmap/ic_launcher',
+    );
+    final initializationSettingsIOS = DarwinInitializationSettings(
       requestAlertPermission: false,
       requestBadgePermission: false,
       requestSoundPermission: false,
+      onDidReceiveLocalNotification: (
+        int? id,
+        String? title,
+        String? body,
+        String? payload,
+      ) async {},
```

Comment:
This callback is empty. Is it possible to remove it so we can make `initializationSettingsIOS` a const again ?

Reply:
This callback is only applicable to iOS versions older than 10. 
I've added a <code>onSelectNotificationOldIOS</code> to this callback. It uses a different set of arguments so I can't reuse <code>onSelectNotification</code>